blog/2021-12-20-cwa-2-15/index.md

```diff
@@ -59,7 +59,7 @@ After the user has selected a certificate and once again given their consent, th
 <br></br>
 
  
-Please notice: the certificates remain on the user's own smartphone and are not stored in the validation system at any time. 
+Please notice: the certificates remain on the user's own smartphone and are not stored in the validation service at any time. 

```

Comment:
Isn't this a contradiction to the sentence above, which says:
"For this purpose, it is temporarily transmitted in encrypted form to the validation service."?